packages/notifi-react-card/lib/components/subscription/EventTypeCustomToggleRow.tsx

```diff
@@ -1,5 +1,12 @@
-import { CustomTopicTypeItem } from '@notifi-network/notifi-frontend-client';
+import {
+  CustomTopicTypeItem,
+  EventTypeItem,
+} from '@notifi-network/notifi-frontend-client';
 import clsx from 'clsx';
+import {
+  subscribeAlertByFrontendClient,
+  unsubscribeAlertByFrontendClient,
+} from 'notifi-react-card/lib/utils/frontendClient';
```

Comment:
i think i prefer to see relative path, like `from '../../utils/frontendClient'`

Reply:
Fixed in PR MVP-2606